python/pfs/utils/coordinates/CoordTransp.py

```diff
@@ -33,22 +42,31 @@ def CoordinateTransform(xyin, za, mode, inr=0., cent=np.array([[0.], [0.]])):
         Transformation mode. Available mode is "sky_pfi", "sky_pfi_hsc"
         "pfi_mcs", "pfi_mcs_wofe", "mcs_pfi"
```

Comment:
Maybe add a sentence like:  This value is unused when `cent` is provided?
Or am I misunderstanding the parameters here?